riscv-sbi.adoc

```diff
@@ -271,6 +271,61 @@ call doesn't return.
 | *RESERVED*                |    0x09-0x0F
 |===
 
+== Power Management Extension, Extension ID: 0x504D (PM)
+
+The Power Management Extension introduces a set of functions that allow the Supervisor to perform per-hart power management operations.
+
+[source, C]
+----
+struct sbiret sbi_pm_hart_up(unsigned long hartid, unsigned long start_addr, unsigned
+long priv)
+----
+
+Brings up *hartid* either during initial boot or after an sbi_hart_down/suspend
+SBI call.
+
+*start_addr* points to a runtime-specified physical address, where a hart can resume execution after its initialization/resume sequence. Before jumping to *start_addr*, the hart MUST configure PMP if present and switch to Supervisor mode.
+
+*priv* is a pointer to a opaque data segment that the caller can use to pass information about
+execution context.
+
+Returns an appropriate SBI error code on sbiret->error.
```

Comment:
nit: Shouldn't the `->` be a `.` instead? If not, beware that markdown will convert the ascii art arrow into a unicode one: "→"

Reply:
Thanks for pointing out. I will change it to sbiret.error